### DISCUSSION TITLE
[feat] option to switch to correct/wrong instead of easy/hard

### DISCUSSION THREAD

**Discussion by mkschulze** [st3v3nmw/obsidian-spaced-repetition#120] [General]
There is some discussion in the spaced repitition community on this matter and it would be nice to have the option, to switch between these modes.

**Comment by mkschulze** [comment]
Here are some thoughts on the algorithms: https://eshapard.github.io/anki/thoughts-on-a-new-algorithm-for-anki.html

Did you implement it this way maybe? Then correct/wrong option would probably not be needed.

**Comment by dzonimn** [comment]
I am the person who was discussing with @mkschulze about this. I commented about how, last I heard (years ago), heavy Anki users (mostly medical students) ran into this phenomenon they called "Ease Hell", where you run into the same card too often leading to inefficiencies. Of course it had been years ago and I don't exactly remember the detailed logic of it, but the takeaway was to avoid using Easy or Hard evaluations and just resort to either Correct or Wrong.

Things might have changed since, but I think the article linked sounds like some kind of improvement to the system, though of course, I can't say I have an intuitive grasp of the implications of such a change.

I noticed that a [modified implementation of Anki's algorithm is being used](https://github.com/st3v3nmw/obsidian-spaced-repetition/wiki/Spaced-Repetition-Algorithm), which suggests to me that perhaps this issue has already been taken into consideration. However, it would be interesting to see if there has been many developments into this discussion since I last read about this being an issue, if it ever was at all.

**Comment by st3v3nmw** [comment]
This plugin's modification changes the default lapse interval change from 0% to 50% (after a lapse, newInterval = oldInterval * lapseIntervalChange).
This kinda handles the ease hell problem; unlike Anki's hard restart, you won't see the card too often.
Coincidentally, I read about that on the blog you linked: https://eshapard.github.io/anki/anki-new-interval-after-a-lapse.html
The plugin also provides a means to reset a card's progress, say after you refresh on the material. The ease factor will get back to the default i.e. 250, instead of painfully working up from low ease factors like 130.
I also read this at the time: https://readbroca.com/anki/ease-hell/

**Comment by mkschulze** [comment]
ok, so this issue can be closed then I guess?

**Comment by st3v3nmw** [comment]
I think so, that is if your concern for ease hell has been addressed well enough.

**Comment by mkschulze** [comment]
Seems good to me.
